src/main/java/frc/robot/subsystems/LedSubsystem.java

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import java.util.Map;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.PWM;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.shuffleboard.SimpleWidget;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.commands.LedToggleCommand;
+
+public class LedSubsystem extends SubsystemBase {
+  private ShuffleboardTab ledTab = Shuffleboard.getTab("Led");
+  private GenericEntry ledState = ledTab.add("OnOff", LedToggleCommand.LedToggle).getEntry();
+  private SimpleWidget ledMode = ledTab.add("Color", LedGPMode).withWidget(BuiltInWidgets.kBooleanBox).withProperties(Map.of("colorWhenFalse", "Purple", "colorWhenTrue", "Yellow"));
+  private PWM pwm;
+  public static boolean LedGPMode = true; //True = Cone, False = Cube
```

Comment:
No public class variables. Also, why not use the enum type you created instead of a boolean?